src/service/uploads/handler.ts

```diff
@@ -341,20 +358,26 @@ function removeExtension(fileName: string): string[] {
   return fileNameWithoutExtension
 }
 
-function normalizeDataType(value: any, type?: string): any {
-  if (type === 'int') {
-    if (!value) {
+function normalizeDataType(value: any, type?: string, def?: any): any {
+  // check for unset value first
+  if (!value) {
+    if (def) {
```

Comment:
this does not work if the default value provided by user is a falsy value in javascript
- number zero `0`
- empty string `""`
- boolean `false`

so I think it is better to write it as
```javascript
if (def !== undefined) {
  // ...
}
```

Reply:
done